website/meta/sidebars.json

```diff
@@ -119,6 +119,7 @@
                     { "text": "Corpus", "url": "/api/corpus" },
                     { "text": "KnowledgeBase", "url": "/api/kb" },
                     { "text": "Lookups", "url": "/api/lookups" },
+                    { "text": "MorphAnalysis", "url": "/api/morphanalysis" },
```

Comment:
I think I ended up grouping the `MorphAnalysis` with `Morphology` because both pages didn't have _that_ much content and they kinda fit together.

Reply:
Oh, okay, I missed that, but I still think I should be able to find a link for it in the sidebar?